seaworthy/tests-core/test_containers.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+
+from docker.models.containers import Container
+
+from seaworthy.checks import dockertest
+from seaworthy.containers import ContainerBase
+from seaworthy.dockerhelper import DockerHelper
+
+IMG = 'nginx:alpine'
+docker_helper = DockerHelper()
+
+
+@dockertest()
+def setUpModule():  # noqa: N802 (The camelCase is mandated by unittest.)
+    docker_helper.setup()
+    docker_helper.pull_image_if_not_found(IMG)
```

Comment:
The `DockerHelper` setup/teardown is cheap, so it would probably be cleaner to put it in `TestContainerBase.setUp()` where we can use `self.addCleanup()` to clean it up. Then the only thing we'd need in `setUpModule()` is image fetching, which can be copied from `test_dockerhelper.py`.